lib/CleantalkSP/SpbctWP/Firewall/WAF.php

```diff
@@ -74,12 +74,25 @@ public function check()
             }
         }
 
+        if ( $_POST || $_GET || $_COOKIE ) {
+            $data_array = array($_POST, $_GET, $_COOKIE);
+        } else {
+            $data_flow_row['data_flow_row'] = file_get_contents('php://input');
```

Comment:
We do need check if file_get_contents does not return false. Also use @ to suppress errors.